task-sdk/src/airflow/sdk/definitions/dag.py

```diff
@@ -557,9 +557,17 @@ def _validate_catchup(self, _, catchup: bool):
             raise ValueError("start_date is required when catchup=True")
 
     @tags.validator
-    def _validate_tags(self, _, tags: Collection[str]):
-        if tags and any(len(tag) > TAG_MAX_LEN for tag in tags):
-            raise ValueError(f"tag cannot be longer than {TAG_MAX_LEN} characters")
+    def _validate_tags(self, _, tags: Collection[str]) -> None:
+        if tags:
```

Comment:
The `if tags:` is pointless here. We didn't have it before, so we can remove it (as a general rule, the less indented the code, the easier it is to read later.